index.html

```diff
@@ -1328,7 +1328,9 @@ <h2>
       </p>
 
       <p>
-The following table summarizes the key formats supported by this specification:
+All public key formats MUST be expressed in either JSON Web Key (JWK)
```

Comment:
@selfissued this is the resolution you requested and that passed on the call earlier in the week. Just drawing your attention to it.